test/app.spec.js

```diff
@@ -97,6 +97,22 @@ describe("App", () => {
   });
 })
 
+describe("Views", () => {
+  it("allows inline styles as object with properties in camelCase", () => {
```

Comment:
Yes, probably the name is not well written. I just want to cover https://github.com/hyperapp/hyperapp/blob/master/app.js#L194 lines...